src/test/java/emissary/test/core/junit5/LogbackTester.java

```diff
@@ -0,0 +1,55 @@
+package emissary.test.core.junit5;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.read.ListAppender;
+import org.apache.commons.lang3.Validate;
+import org.slf4j.LoggerFactory;
+
+import java.io.Closeable;
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LogbackTester implements Closeable {
+    public final String name;
+    public final Logger logger;
+    public final ListAppender<ILoggingEvent> appender;
+
+    public LogbackTester(final String name) {
+        Validate.notNull(name, "Required: name != null");
+
+        this.name = name;
+        logger = (Logger) LoggerFactory.getLogger(name);
+        appender = new ListAppender<>();
+
+        appender.setContext(logger.getLoggerContext());
+        appender.start();
+        logger.addAppender(appender);
+        logger.setAdditive(false);
+    }
+
+    public void checkLogList(final Level[] levels, final String[] messages, final boolean[] throwables) {
```

Comment:
Maybe a future refactor, but this seems like an awkward interface. If we were to define a new class - say, `SimplifedLogEvent` - this method instead could take a `List<SimplifiedLogEvent>` and could eliminate all or most of the `Validate.xxx` calls 

Maybe something similar to...
```java
static class SimplifiedLogEvent {
   final Level level;
   final String message;
   final Optional<? extends Throwable> throwable;

   SimplifiedLogEvent (Level level, String message, @Nullable Throwable throwable) {
      ...
   }
  
}
```